mlrun/feature_store/api.py

```diff
@@ -662,6 +662,9 @@ def preview(
     :param sample_size:    num of rows to sample from the dataset (for large datasets)
     """
     # preview reads the source as a pandas df, which is not fully compatible with spark
```

Comment:
this comment should be right above the `if featureset.spec.engine == "spark":`

Reply:
Ty!